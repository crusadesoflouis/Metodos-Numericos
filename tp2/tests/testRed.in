../datosEntrenamiento/ImagenesCarasRed//s1/1.pgm, 1,
../datosEntrenamiento/ImagenesCarasRed//s1/2.pgm, 1,
../datosEntrenamiento/ImagenesCarasRed//s1/3.pgm, 1,
../datosEntrenamiento/ImagenesCarasRed//s1/4.pgm, 1,
../datosEntrenamiento/ImagenesCarasRed//s1/5.pgm, 1,
../datosEntrenamiento/ImagenesCarasRed//s2/1.pgm, 2,
../datosEntrenamiento/ImagenesCarasRed//s2/2.pgm, 2,
../datosEntrenamiento/ImagenesCarasRed//s2/3.pgm, 2,
../datosEntrenamiento/ImagenesCarasRed//s2/4.pgm, 2,
../datosEntrenamiento/ImagenesCarasRed//s2/5.pgm, 2,
../datosEntrenamiento/ImagenesCarasRed//s3/1.pgm, 3,
../datosEntrenamiento/ImagenesCarasRed//s3/2.pgm, 3,
../datosEntrenamiento/ImagenesCarasRed//s3/3.pgm, 3,
../datosEntrenamiento/ImagenesCarasRed//s3/4.pgm, 3,
../datosEntrenamiento/ImagenesCarasRed//s3/5.pgm, 3,
../datosEntrenamiento/ImagenesCarasRed//s4/1.pgm, 4,
../datosEntrenamiento/ImagenesCarasRed//s4/2.pgm, 4,
../datosEntrenamiento/ImagenesCarasRed//s4/3.pgm, 4,
../datosEntrenamiento/ImagenesCarasRed//s4/4.pgm, 4,
../datosEntrenamiento/ImagenesCarasRed//s4/5.pgm, 4,
../datosEntrenamiento/ImagenesCarasRed//s5/1.pgm, 5,
../datosEntrenamiento/ImagenesCarasRed//s5/2.pgm, 5,
../datosEntrenamiento/ImagenesCarasRed//s5/3.pgm, 5,
../datosEntrenamiento/ImagenesCarasRed//s5/4.pgm, 5,
../datosEntrenamiento/ImagenesCarasRed//s5/5.pgm, 5,
../datosEntrenamiento/ImagenesCarasRed//s6/1.pgm, 6,
../datosEntrenamiento/ImagenesCarasRed//s6/2.pgm, 6,
../datosEntrenamiento/ImagenesCarasRed//s6/3.pgm, 6,
../datosEntrenamiento/ImagenesCarasRed//s6/4.pgm, 6,
../datosEntrenamiento/ImagenesCarasRed//s6/5.pgm, 6,
../datosEntrenamiento/ImagenesCarasRed//s7/1.pgm, 7,
../datosEntrenamiento/ImagenesCarasRed//s7/2.pgm, 7,
../datosEntrenamiento/ImagenesCarasRed//s7/3.pgm, 7,
../datosEntrenamiento/ImagenesCarasRed//s7/4.pgm, 7,
../datosEntrenamiento/ImagenesCarasRed//s7/5.pgm, 7,
../datosEntrenamiento/ImagenesCarasRed//s8/1.pgm, 8,
../datosEntrenamiento/ImagenesCarasRed//s8/2.pgm, 8,
../datosEntrenamiento/ImagenesCarasRed//s8/3.pgm, 8,
../datosEntrenamiento/ImagenesCarasRed//s8/4.pgm, 8,
../datosEntrenamiento/ImagenesCarasRed//s8/5.pgm, 8,
../datosEntrenamiento/ImagenesCarasRed//s9/1.pgm, 9,
../datosEntrenamiento/ImagenesCarasRed//s9/2.pgm, 9,
../datosEntrenamiento/ImagenesCarasRed//s9/3.pgm, 9,
../datosEntrenamiento/ImagenesCarasRed//s9/4.pgm, 9,
../datosEntrenamiento/ImagenesCarasRed//s9/5.pgm, 9,
../datosEntrenamiento/ImagenesCarasRed//s10/1.pgm, 10,
../datosEntrenamiento/ImagenesCarasRed//s10/2.pgm, 10,
../datosEntrenamiento/ImagenesCarasRed//s10/3.pgm, 10,
../datosEntrenamiento/ImagenesCarasRed//s10/4.pgm, 10,
../datosEntrenamiento/ImagenesCarasRed//s10/5.pgm, 10,
../datosEntrenamiento/ImagenesCarasRed//s11/1.pgm, 11,
../datosEntrenamiento/ImagenesCarasRed//s11/2.pgm, 11,
../datosEntrenamiento/ImagenesCarasRed//s11/3.pgm, 11,
../datosEntrenamiento/ImagenesCarasRed//s11/4.pgm, 11,
../datosEntrenamiento/ImagenesCarasRed//s11/5.pgm, 11,
../datosEntrenamiento/ImagenesCarasRed//s12/1.pgm, 12,
../datosEntrenamiento/ImagenesCarasRed//s12/2.pgm, 12,
../datosEntrenamiento/ImagenesCarasRed//s12/3.pgm, 12,
../datosEntrenamiento/ImagenesCarasRed//s12/4.pgm, 12,
../datosEntrenamiento/ImagenesCarasRed//s12/5.pgm, 12,
../datosEntrenamiento/ImagenesCarasRed//s13/1.pgm, 13,
../datosEntrenamiento/ImagenesCarasRed//s13/2.pgm, 13,
../datosEntrenamiento/ImagenesCarasRed//s13/3.pgm, 13,
../datosEntrenamiento/ImagenesCarasRed//s13/4.pgm, 13,
../datosEntrenamiento/ImagenesCarasRed//s13/5.pgm, 13,
../datosEntrenamiento/ImagenesCarasRed//s14/1.pgm, 14,
../datosEntrenamiento/ImagenesCarasRed//s14/2.pgm, 14,
../datosEntrenamiento/ImagenesCarasRed//s14/3.pgm, 14,
../datosEntrenamiento/ImagenesCarasRed//s14/4.pgm, 14,
../datosEntrenamiento/ImagenesCarasRed//s14/5.pgm, 14,
../datosEntrenamiento/ImagenesCarasRed//s15/1.pgm, 15,
../datosEntrenamiento/ImagenesCarasRed//s15/2.pgm, 15,
../datosEntrenamiento/ImagenesCarasRed//s15/3.pgm, 15,
../datosEntrenamiento/ImagenesCarasRed//s15/4.pgm, 15,
../datosEntrenamiento/ImagenesCarasRed//s15/5.pgm, 15,
../datosEntrenamiento/ImagenesCarasRed//s16/1.pgm, 16,
../datosEntrenamiento/ImagenesCarasRed//s16/2.pgm, 16,
../datosEntrenamiento/ImagenesCarasRed//s16/3.pgm, 16,
../datosEntrenamiento/ImagenesCarasRed//s16/4.pgm, 16,
../datosEntrenamiento/ImagenesCarasRed//s16/5.pgm, 16,
../datosEntrenamiento/ImagenesCarasRed//s17/1.pgm, 17,
../datosEntrenamiento/ImagenesCarasRed//s17/2.pgm, 17,
../datosEntrenamiento/ImagenesCarasRed//s17/3.pgm, 17,
../datosEntrenamiento/ImagenesCarasRed//s17/4.pgm, 17,
../datosEntrenamiento/ImagenesCarasRed//s17/5.pgm, 17,
../datosEntrenamiento/ImagenesCarasRed//s18/1.pgm, 18,
../datosEntrenamiento/ImagenesCarasRed//s18/2.pgm, 18,
../datosEntrenamiento/ImagenesCarasRed//s18/3.pgm, 18,
../datosEntrenamiento/ImagenesCarasRed//s18/4.pgm, 18,
../datosEntrenamiento/ImagenesCarasRed//s18/5.pgm, 18,
../datosEntrenamiento/ImagenesCarasRed//s19/1.pgm, 19,
../datosEntrenamiento/ImagenesCarasRed//s19/2.pgm, 19,
../datosEntrenamiento/ImagenesCarasRed//s19/3.pgm, 19,
../datosEntrenamiento/ImagenesCarasRed//s19/4.pgm, 19,
../datosEntrenamiento/ImagenesCarasRed//s19/5.pgm, 19,
../datosEntrenamiento/ImagenesCarasRed//s20/1.pgm, 20,
../datosEntrenamiento/ImagenesCarasRed//s20/2.pgm, 20,
../datosEntrenamiento/ImagenesCarasRed//s20/3.pgm, 20,
../datosEntrenamiento/ImagenesCarasRed//s20/4.pgm, 20,
../datosEntrenamiento/ImagenesCarasRed//s20/5.pgm, 20,
../datosEntrenamiento/ImagenesCarasRed//s21/1.pgm, 21,
../datosEntrenamiento/ImagenesCarasRed//s21/2.pgm, 21,
../datosEntrenamiento/ImagenesCarasRed//s21/3.pgm, 21,
../datosEntrenamiento/ImagenesCarasRed//s21/4.pgm, 21,
../datosEntrenamiento/ImagenesCarasRed//s21/5.pgm, 21,
../datosEntrenamiento/ImagenesCarasRed//s22/1.pgm, 22,
../datosEntrenamiento/ImagenesCarasRed//s22/2.pgm, 22,
../datosEntrenamiento/ImagenesCarasRed//s22/3.pgm, 22,
../datosEntrenamiento/ImagenesCarasRed//s22/4.pgm, 22,
../datosEntrenamiento/ImagenesCarasRed//s22/5.pgm, 22,
../datosEntrenamiento/ImagenesCarasRed//s23/1.pgm, 23,
../datosEntrenamiento/ImagenesCarasRed//s23/2.pgm, 23,
../datosEntrenamiento/ImagenesCarasRed//s23/3.pgm, 23,
../datosEntrenamiento/ImagenesCarasRed//s23/4.pgm, 23,
../datosEntrenamiento/ImagenesCarasRed//s23/5.pgm, 23,
../datosEntrenamiento/ImagenesCarasRed//s24/1.pgm, 24,
../datosEntrenamiento/ImagenesCarasRed//s24/2.pgm, 24,
../datosEntrenamiento/ImagenesCarasRed//s24/3.pgm, 24,
../datosEntrenamiento/ImagenesCarasRed//s24/4.pgm, 24,
../datosEntrenamiento/ImagenesCarasRed//s24/5.pgm, 24,
../datosEntrenamiento/ImagenesCarasRed//s25/1.pgm, 25,
../datosEntrenamiento/ImagenesCarasRed//s25/2.pgm, 25,
../datosEntrenamiento/ImagenesCarasRed//s25/3.pgm, 25,
../datosEntrenamiento/ImagenesCarasRed//s25/4.pgm, 25,
../datosEntrenamiento/ImagenesCarasRed//s25/5.pgm, 25,
../datosEntrenamiento/ImagenesCarasRed//s26/1.pgm, 26,
../datosEntrenamiento/ImagenesCarasRed//s26/2.pgm, 26,
../datosEntrenamiento/ImagenesCarasRed//s26/3.pgm, 26,
../datosEntrenamiento/ImagenesCarasRed//s26/4.pgm, 26,
../datosEntrenamiento/ImagenesCarasRed//s26/5.pgm, 26,
../datosEntrenamiento/ImagenesCarasRed//s27/1.pgm, 27,
../datosEntrenamiento/ImagenesCarasRed//s27/2.pgm, 27,
../datosEntrenamiento/ImagenesCarasRed//s27/3.pgm, 27,
../datosEntrenamiento/ImagenesCarasRed//s27/4.pgm, 27,
../datosEntrenamiento/ImagenesCarasRed//s27/5.pgm, 27,
../datosEntrenamiento/ImagenesCarasRed//s28/1.pgm, 28,
../datosEntrenamiento/ImagenesCarasRed//s28/2.pgm, 28,
../datosEntrenamiento/ImagenesCarasRed//s28/3.pgm, 28,
../datosEntrenamiento/ImagenesCarasRed//s28/4.pgm, 28,
../datosEntrenamiento/ImagenesCarasRed//s28/5.pgm, 28,
../datosEntrenamiento/ImagenesCarasRed//s29/1.pgm, 29,
../datosEntrenamiento/ImagenesCarasRed//s29/2.pgm, 29,
../datosEntrenamiento/ImagenesCarasRed//s29/3.pgm, 29,
../datosEntrenamiento/ImagenesCarasRed//s29/4.pgm, 29,
../datosEntrenamiento/ImagenesCarasRed//s29/5.pgm, 29,
../datosEntrenamiento/ImagenesCarasRed//s30/1.pgm, 30,
../datosEntrenamiento/ImagenesCarasRed//s30/2.pgm, 30,
../datosEntrenamiento/ImagenesCarasRed//s30/3.pgm, 30,
../datosEntrenamiento/ImagenesCarasRed//s30/4.pgm, 30,
../datosEntrenamiento/ImagenesCarasRed//s30/5.pgm, 30,
../datosEntrenamiento/ImagenesCarasRed//s31/1.pgm, 31,
../datosEntrenamiento/ImagenesCarasRed//s31/2.pgm, 31,
../datosEntrenamiento/ImagenesCarasRed//s31/3.pgm, 31,
../datosEntrenamiento/ImagenesCarasRed//s31/4.pgm, 31,
../datosEntrenamiento/ImagenesCarasRed//s31/5.pgm, 31,
../datosEntrenamiento/ImagenesCarasRed//s32/1.pgm, 32,
../datosEntrenamiento/ImagenesCarasRed//s32/2.pgm, 32,
../datosEntrenamiento/ImagenesCarasRed//s32/3.pgm, 32,
../datosEntrenamiento/ImagenesCarasRed//s32/4.pgm, 32,
../datosEntrenamiento/ImagenesCarasRed//s32/5.pgm, 32,
../datosEntrenamiento/ImagenesCarasRed//s33/1.pgm, 33,
../datosEntrenamiento/ImagenesCarasRed//s33/2.pgm, 33,
../datosEntrenamiento/ImagenesCarasRed//s33/3.pgm, 33,
../datosEntrenamiento/ImagenesCarasRed//s33/4.pgm, 33,
../datosEntrenamiento/ImagenesCarasRed//s33/5.pgm, 33,
../datosEntrenamiento/ImagenesCarasRed//s34/1.pgm, 34,
../datosEntrenamiento/ImagenesCarasRed//s34/2.pgm, 34,
../datosEntrenamiento/ImagenesCarasRed//s34/3.pgm, 34,
../datosEntrenamiento/ImagenesCarasRed//s34/4.pgm, 34,
../datosEntrenamiento/ImagenesCarasRed//s34/5.pgm, 34,
../datosEntrenamiento/ImagenesCarasRed//s35/1.pgm, 35,
../datosEntrenamiento/ImagenesCarasRed//s35/2.pgm, 35,
../datosEntrenamiento/ImagenesCarasRed//s35/3.pgm, 35,
../datosEntrenamiento/ImagenesCarasRed//s35/4.pgm, 35,
../datosEntrenamiento/ImagenesCarasRed//s35/5.pgm, 35,
../datosEntrenamiento/ImagenesCarasRed//s36/1.pgm, 36,
../datosEntrenamiento/ImagenesCarasRed//s36/2.pgm, 36,
../datosEntrenamiento/ImagenesCarasRed//s36/3.pgm, 36,
../datosEntrenamiento/ImagenesCarasRed//s36/4.pgm, 36,
../datosEntrenamiento/ImagenesCarasRed//s36/5.pgm, 36,
../datosEntrenamiento/ImagenesCarasRed//s37/1.pgm, 37,
../datosEntrenamiento/ImagenesCarasRed//s37/2.pgm, 37,
../datosEntrenamiento/ImagenesCarasRed//s37/3.pgm, 37,
../datosEntrenamiento/ImagenesCarasRed//s37/4.pgm, 37,
../datosEntrenamiento/ImagenesCarasRed//s37/5.pgm, 37,
../datosEntrenamiento/ImagenesCarasRed//s38/1.pgm, 38,
../datosEntrenamiento/ImagenesCarasRed//s38/2.pgm, 38,
../datosEntrenamiento/ImagenesCarasRed//s38/3.pgm, 38,
../datosEntrenamiento/ImagenesCarasRed//s38/4.pgm, 38,
../datosEntrenamiento/ImagenesCarasRed//s38/5.pgm, 38,
../datosEntrenamiento/ImagenesCarasRed//s39/1.pgm, 39,
../datosEntrenamiento/ImagenesCarasRed//s39/2.pgm, 39,
../datosEntrenamiento/ImagenesCarasRed//s39/3.pgm, 39,
../datosEntrenamiento/ImagenesCarasRed//s39/4.pgm, 39,
../datosEntrenamiento/ImagenesCarasRed//s39/5.pgm, 39,
../datosEntrenamiento/ImagenesCarasRed//s40/1.pgm, 40,
../datosEntrenamiento/ImagenesCarasRed//s40/2.pgm, 40,
../datosEntrenamiento/ImagenesCarasRed//s40/3.pgm, 40,
../datosEntrenamiento/ImagenesCarasRed//s40/4.pgm, 40,
../datosEntrenamiento/ImagenesCarasRed//s40/5.pgm, 40,
../datosEntrenamiento/ImagenesCarasRed//s41/1.pgm, 41,
../datosEntrenamiento/ImagenesCarasRed//s41/2.pgm, 41,
../datosEntrenamiento/ImagenesCarasRed//s41/3.pgm, 41,
../datosEntrenamiento/ImagenesCarasRed//s41/4.pgm, 41,
../datosEntrenamiento/ImagenesCarasRed//s41/5.pgm, 41,