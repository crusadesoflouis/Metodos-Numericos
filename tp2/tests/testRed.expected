440.417013
363.145784
279.734131
238.139060
226.984990
193.166470
174.347739
161.672764
147.812135
129.977376
128.206632
118.757460
115.197549
107.645970
106.860620
